ts/importFlake.ts

```diff
@@ -0,0 +1,63 @@
+import { Check } from "./check.ts";
+import { Environment, mkEnvironment } from "./environment.ts";
+import { Executable, mkExecutable } from "./executable.ts";
+import { NixExpression, nixFlakeDep, nixRaw, nixStrLit } from "./nix.ts";
+import { mkPackage, Package } from "./package.ts";
+
+function getFlake(flakeDir: string): NixExpression {
+  if (flakeDir.match(/^[.][.]?[/]/)) {
```

Comment:
This matches relative and absolute paths?

Btw, there's also lots of other flake url syntax options, see [here](https://nixos.org/manual/nix/unstable/command-ref/new-cli/nix3-flake#url-like-syntax) and [here](https://nixos.org/manual/nix/unstable/command-ref/new-cli/nix3-flake#types) lists all possible types. So e.g. `file:...` and `path:...` are valid flake urls. I don't think we have to support any of this besides what's currently implemented. But 1. I wanted you to know those exist and 2. maybe we should think about what happens with those right now.

Reply:
This matches relative paths only at the moment. Do you think we should support absolute paths? One thing I realized in testing is that the only thing that would break when using a flake input vs callFlake is parent path imports. Maybe we can experiment with `file:...` and `path:...` tomorrow and see how they interact with both flake inputs and callFlake